go.mod

```diff
@@ -26,101 +26,105 @@ require (
 	google.golang.org/protobuf v1.27.1
 	gopkg.in/resty.v1 v1.12.0
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
-	k8s.io/kubernetes v1.23.4
-	k8s.io/utils v0.0.0-20211208161948-7d6a63dca704
-	sigs.k8s.io/controller-runtime v0.11.1
-)
-
-require (
-	github.com/apache/tinkerpop/gremlin-go v0.0.0-20220530191148-29272fa563ec
```

Comment:
Not sure why there was a dedicated require block for this, I moved it to the first one above.

Reply:
Go does not do single block, it can do one per dependency run also.